### ISSUE TITLE
manage commands

### ISSUE THREAD

**Issue by lenny76** [yagop/node-telegram-bot-api#28]
hi, 
what is the best way to manage list of commands and replies?
now I'm filling the app with a lot of 'if'...
if command = 'xxx' {}
if command = 'xxx' {}
if command = 'xxx' {}

is there a better way, so I can manage easily the app?
thanks

**Comment by yorkcommatyler** [comment]
I actually had a similar question: how do you manage commands at all? I'm not finding anything in either the telegram docs or the node-telegram-bot-api source for it.

**Comment by yagop** [comment]
I implemented a new feature on new version (0.15.0) to work around this.

```js
bot.onText(/\/echo (.+)/, function (msg, match) {
  var fromId = msg.from.id;
  var resp = match[1];
  bot.sendMessage(fromId, resp);
});
```

**Comment by lenny76** [comment]
+1
